cmd/ocm-support/registryCredentials/delete/cmd.go

```diff
@@ -0,0 +1,99 @@
+package delete
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"ocm-support-cli/pkg/account"
+	"ocm-support-cli/pkg/registry_credential"
+)
+
+var args struct {
+	all bool
+}
+
+// Cmd ...
+var Cmd = &cobra.Command{
+	Use:   "delete accountID registryCredentialID",
+	Short: "Deletes registry credentials of the given ID.",
+	Long:  "Deletes registry credentials of the given ID.",
+	RunE:  run,
+	Args:  cobra.MinimumNArgs(1),
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.all,
+		"all",
+		false,
+		"If true, deletes all registry credentials for the given account ID.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) error {
+	if len(argv) < 1 {
+		return fmt.Errorf("expected at least one argument")
+	}
+
+	accountID := argv[0]
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+
+	account, err := account.GetAccount(accountID, connection)
```

Comment:
Variable 'account' collides with imported package name

Reply:
Changed the variable to requestedAccount